fix(a11y): final fix for WCAG 1.1.1 image alt should describe image content accurately (#17)
scottrigby/basekamp-next: content/events/congress-of-collectives.md

title: Congress of the Collectives
images:
  - src: p1040553.jpg

>     alt: Group of people seated and standing at long dining table sharing food and laptops in colorfully lit room

Reviewer: ```suggestion
    alt: Group of people seated at long dining table sharing food with laptops open
```
